package-lock.json

```diff
@@ -1,13 +1,13 @@
 {
   "name": "@okta/okta-signin-widget",
-  "version": "2.5.0",
+  "version": "2.6.0",
   "lockfileVersion": 1,
   "requires": true,
   "dependencies": {
     "@okta/okta-auth-js": {
-      "version": "1.11.0",
-      "resolved": "https://registry.npmjs.org/@okta/okta-auth-js/-/okta-auth-js-1.11.0.tgz",
-      "integrity": "sha1-k6/QIE1w1vasBCBmfa/Ff156j/4=",
+      "version": "1.13.0",
+      "resolved": "https://artifacts.aue1d.saasure.com:443/artifactory/api/npm/npm-okta/@okta/okta-auth-js/-/@okta/okta-auth-js-1.13.0.tgz",
```

Comment:
This has to be the public npm registry, otherwise people that do not have access to our internal artifactory won't be able to download auth-js